### PR TITLE
1039/paid hourly

### PR DESCRIPTION
**Pull request by nabil-tazi**
I addressed the following items from Notion Ticket #1039 :
1: (offer page) Payment value was wrong
2: (offer page) Paid Hourly rate was missing
5: (payment page) "mission" word replaced by "job"
6: (payment page) “Hourly submissions” replaced by “Hours submissions”
7: (payment page) Fixed the data in the Hours submission section (real submissions are displayed)
8: (payment page) Payment summary is adapted to Hourly-paid jobs
9: (complete mission page) Connected the submissions to the real data from the API
10: (Submit hours) Fixed the payload when submitting hours, added UI to select the week according to Figma
11: (Confirm hours) The organization can confirm the hours submissions

### CODE REVIEW

File: src/components/molecules/applicant-list-pay/applicant-list-pay.tsx [hunk 1]
```diff
@@ -74,8 +88,14 @@ export const ApplicantListPay = (props: ApplicantListPayProps): JSX.Element => {
             </div>,
             props.isPaidCrypto
           )}
-          {printWhen(confirmBtn(applicant.id, applicant.payment?.meta?.id), props.confirmable || applicant.status === 'COMPLETE')}
-          {printWhen(feedbackBtn(applicant.id, applicant.status), !!props?.onFeedback && applicant?.user_feedback === null)}
+          {printWhen(
```
Comment (ImanRezasefat) [inline src/components/molecules/applicant-list-pay/applicant-list-pay.tsx:91]: pls setup your prettier

File: public/icons/edit.svg
Comment (ImanRezasefat) [inline public/icons/edit.svg:1]: we already have this in project

File: environments/.env.development [hunk 2]
```diff
@@ -4,5 +4,4 @@ VITE_SOCKET_URL=https://dev.socious.io
 VITE_ACCESS_EXPIRE=0.0104166666666667
 VITE_REFRESH_EXPIRE=30
 VITE_JP_STRIPE_PUBLIC_KEY=pk_test_51L8MQWFiHSKRe5D1xc9fIXRnTQ5IEDchTQjebOhoggYQadoQWjOc5wy6RBWifN7RjdN3Ixg7i83mYW0E3MZ8kuCK00Ra4PEPQ6
```
Comment (ImanRezasefat) [inline environments/.env.development:6]: we removed .env.development pls remove the file and use .env file instead

File: src/components/templates/top-up-summary-card/top-up-summary-card.types.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { PaymentDetailProps } from 'src/components/molecules/payment-detail/payment-detail.types';
+
+export interface TopUpSummaryCardProps {
+  //extends PaymentDetailProps {
```
Comment (ImanRezasefat) [inline src/components/templates/top-up-summary-card/top-up-summary-card.types.ts:4]: pls avoid pushing commented code

File: src/pages/complete-mission/mobile/mobile.tsx [hunk 4]
```diff
@@ -60,14 +81,72 @@ export const Mobile = (): JSX.Element => {
...
+    <Accordion title="Hours submission" id="hours-submission">
+      <div className={css.missionDetailContainer}>{printWhen(SubmitHoursJSX(), submit_hours.length > 0)}</div>
+    </Accordion>
+  );
   return (
     <TopFixedMobile>
       <Header title={`${offer.job_category.name}`} onBack={() => history.back()} />
       <div className={css.body}>
         {printWhen(offeredMessageBoxJSX, status === 'HIRED')}
+        {/*  TODO show warning submit hours*/}
```
Comment (ImanRezasefat) [inline src/pages/complete-mission/mobile/mobile.tsx:145]: pls remove the todos

File: src/pages/job-apply/apply/mobile/mobile.tsx [hunk 5]
```diff
@@ -10,8 +10,11 @@ import { ExpandableText } from 'src/components/atoms/expandable-text';
 import { printWhen } from 'src/core/utils';
 import { useApplyShared } from '../apply.shared';
```
Comment (ImanRezasefat) [inline src/pages/job-apply/apply/mobile/mobile.tsx:11]: We are avoiding having different pages for mobile and desktop. Can you refactor your code to have 1 page for both mobile and desktop?pls checkout latest merge requests.

File: src/pages/job-apply/my-jobs/my-jobs.services.tsx [hunk 6]
```diff
@@ -117,6 +120,28 @@ function setStatusJSX(status: keyof typeof APPLICANT_STATUS) {
...
+    alignItems: 'center',
+    fontSize: 'var(--font-size-s)',
+  };
+  const completedStyles: CSSProperties = {
+    padding: '10px',
+    height: '1.5rem',
+    backgroundColor: 'var(--color-gray-04)',
+    borderRadius: '25px',
+    display: 'flex',
+    color: '#333',
```
Comment (ImanRezasefat) [inline src/pages/job-apply/my-jobs/my-jobs.services.tsx:140]: pls avoid using hard-coded colors and use variables(add if it doesn't exist)

File: src/pages/job-create/final-review/review-modal/index.tsx [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+import { AlertModal } from "src/components/organisms/alert-modal";
+import { translateExperienceLevel } from 'src/constants/EXPERIENCE_LEVEL';
+import { translatePaymentType } from 'src/constants/PROJECT_PAYMENT_TYPE';
+import { translateProjectType } from 'src/constants/PROJECT_TYPES';
+import { getCountryByShortname } from 'src/constants/COUNTRIES';
+import { translateProjectLength } from 'src/constants/PROJECT_LENGTH';
+import { translatePaymentRange } from 'src/constants/PAYMENT_RANGE';
+import { translateRemotePreferences } from 'src/constants/PROJECT_REMOTE_PREFERENCE';
+import { convertSnakeCaseToLowerCase } from 'src/core/stringTransformation';
+import {getCausesList} from "../../../../components/organisms/job-list/job-list.services";
```
Comment (ImanRezasefat) [inline src/pages/job-create/final-review/review-modal/index.tsx:20]: pls use absolute path

File: src/pages/job-create/screener-questions/mobile/mobile.tsx [hunk 8]
```diff
@@ -27,11 +27,11 @@ export const Mobile: React.FC = () => {
     useScreenerQuestionsShared();
 
   function submitSkip() {
-    dialog.alert({ title: 'Successfully', message: 'You have successfully created a job post' }).then(() => {
-      navigate({ to: `/m/jobs/created/${formState.question_project_id.identity_id}` });
-      store.dispatch(resetCreatedQuestion());
-      store.dispatch(resetCreatePostWizard());
-    });
+    // dialog.alert({ title: 'Successfully', message: 'You have successfully created a job post' }).then(() => {
```
Comment (ImanRezasefat) [inline src/pages/job-create/screener-questions/mobile/mobile.tsx:30]: pls remove unused function

File: src/pages/job-apply/apply/mobile/mobile.tsx (hunk 5 above)
Comment (nabil-tazi) [reply src/pages/job-apply/apply/mobile/mobile.tsx:11]: I wasn't aware of this when I was developing. Can we accept this and refactor it later along with the other pages?

File: src/pages/job-apply/my-jobs/my-jobs.services.tsx (hunk 6 above)
Comment (nabil-tazi) [reply src/pages/job-apply/my-jobs/my-jobs.services.tsx]: This was legacy from mohammadali branch. I fixed replaced the hardcoded colors with variables.

**Review by ImanRezasefat — APPROVED** [review]

**Pull request closed without merging**